src/css/doc.css

```diff
@@ -512,11 +513,15 @@ ul ul ul {
 .doc blockquote * {
   font-weight: inherit;
 }
-/* .doc .abstract blockquote::before {
-  content: "Summary: ";
+/* 
+ .doc .abstract blockquote::before {
+  content: "Quick Summary \a";
```

Comment:
I was trying to see where "Quick Summary" showed up... OK, it's commented out.
I'm happy for this to be deleted, but I take the point that we're not CSS experts, and so having a previously working-ish example of a `::before` pseudo-element is worth something.

Might be worth *adding* a comment describing what this block could be used for, e.g. reasons to uncomment it in future?

Reply:
I can do this thing.